controllers/presentationController.js

```diff
@@ -11,31 +11,69 @@ async function getAllPresentations(req, res) {
     if (!allPresentations) {
       res
         .status(404)
-        .send({ result: "error", error: "Presentations not found" });
+        .json({ message: "No presentations found created by the user" });
       return;
     }
 
     res.json(allPresentations);
   } catch (err) {
-    res.status(500).send({ result: "error", error: "Internal Server Error" });
+    res.status(500).json({ result: "error", error: "Internal Server Error" });
```

Comment:
```suggestion
    next(err);
```
app.js에 있는 에러 핸들러 미들웨어가 에러 처리를 하도록 구현해야 될 것 같습니다